eng/common/scripts/Git-Repo-Sync.ps1

```diff
@@ -0,0 +1,119 @@
+ #!/usr/bin/env pwsh -c
+
+<#
+.DESCRIPTION
+Sync all files from SourceRepo to TargetRepo directlly.
+
+.PARAMETER SourceRepo
+The GitHub repository that needs to be referenced.
+
+.PARAMETER SourceBranch
+The branch of Source GitHub repository that needs to be referenced.
+
+.PARAMETER TargetRepo
+The GitHub repository will synced.
+
+.PARAMETER TargetBranch
+The branch of Target GitHub repository will synced.
+
+.PARAMETER AuthToken
+A personal access token
+
+.PARAMETER Rebase
+Keep the commit record when syning.
+#>
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$SourceRepo,
+
+  [Parameter(Mandatory = $false)]
+  [string]$SourceBranch,
+
+  [Parameter(Mandatory = $true)]
+  [string]$TargetRepo,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TargetBranch,
+
+  [Parameter(Mandatory = $true)]
+  [string]$AuthToken,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Rebase,
+  
+  $user="azure-sdk",
```

Comment:
Unless we have usages where we want to change these values I'd suggest excluding them for now.

Reply:
These parameters have been removed to the external